src/sentry/models/project.py

```diff
@@ -324,6 +327,19 @@ def transfer_to(self, team=None, organization=None):
         if team is not None:
             self.add_team(team)
 
+        # Remove alert owners not in new org
+        alert_rules = AlertRule.objects.fetch_for_project(self).filter(owner_id__isnull=False)
```

Comment:
I'm not sure how well these actually transfer... since these alert rules exist at the org level and just have a project associated with them. Since in practice we only ever have the associated with one project now, maybe we should update the org they belong to as well?